client/human/HumanClientApp.cpp

```diff
@@ -1127,3 +1128,44 @@ HumanClientApp* HumanClientApp::GetApp()
 
 void HumanClientApp::Initialize()
 {}
+
+void HumanClientApp::OpenURL(const std::string& url) {
```

Comment:
I'm 100% sure that there are trusted libraries for checking if a string is a valid URL, and 110% sure that there are more constraints on URLs than you've tested for here.